### PR TITLE
f-content-cards@1.11.0 🛃 First Time Customer card

### PR DESCRIPTION
**Pull request by minrwhite**
## Added
- `Restaurant FTC Offer Card` card type, with associated tests and story.

---

### Styling for UK:

image

### Styling for AU/NZ (and others, though FTC not used)

image

- [ ] README and/or UI Documentation has been [created|updated]
- [x] Unit tests have been [created|updated]
- [x] This code has been checked with regard to [our accessibility standards](http://fozzie.just-eat.com/documentation/general/accessibility/checklist)

## Browsers Tested

- [x] Chrome (latest)
- [x] Internet Explorer 11
- [ ] Mobile (Please list device/browser – Ideally one iPhone model and one Android)

### CODE REVIEW

File: packages/f-content-cards/stories/FirstTimeCustomerCard.stories.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+ *
+ * @param {Object} args
+ * @param {Object}
+ * @return {{
+ *  template: string,
+ *  components: {Object}
+ *  props: {Object}
+ *  provide: {Function}
+ * }}
+ * @return {{
```
Comment (DamianMullins) [inline packages/f-content-cards/stories/FirstTimeCustomerCard.stories.js:22]: Do you need the two `@return`s here?

Comment (minrwhite) [reply packages/f-content-cards/stories/FirstTimeCustomerCard.stories.js:22]: Good spot, fixed 👍

File: packages/f-content-cards/src/components/cardTemplates/FirstTimeCustomerCard.vue [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+                <span
+                    :class="['c-restaurantCard-banner-content', $style['c-restaurantCard-banner-content'], {
+                        [$style['c-restaurantCard-banner-content-legacy']]: usesLegacyStyles
+                    }]"
+                    data-test-id="FirstTimeCustomerCard-Banner">
+                    {{ card.banner }}
+                </span>
+            </span>
+            <template v-if="!bannerBeforeDescription">
+                for first time user
```
Comment (DamianMullins) [inline packages/f-content-cards/src/components/cardTemplates/FirstTimeCustomerCard.vue:20]: Does this need to be localised?

File: packages/f-content-cards/stories/ContentCards.stories.js [hunk 3]
```diff
@@ -113,4 +113,4 @@ export function ContentCardscomponent () {
     };
 }
 
-ContentCardscomponent.storyName = 'f-content-cards';
+ContentCardsComponent.storyName = 'f-content-cards';
```
Comment (DamianMullins) [inline packages/f-content-cards/stories/ContentCards.stories.js:116]: 🚀

File: packages/f-content-cards/src/components/cardTemplates/FirstTimeCustomerCard.vue (hunk 2 above)
Comment (minrwhite) [reply packages/f-content-cards/src/components/cardTemplates/FirstTimeCustomerCard.vue:20]: Unfortunately this is (and needs to be) a verbtim copy of what we have in HomeWeb (which was never localised, either). Currently the FTC card is only offered in AU/NZ so localising to other tenants is not an issue.

**Review by DamianMullins — APPROVED** [review]

**Review by asherstoppard — APPROVED** [review]

**Pull request merged**